docs/notebooks/MetaOT.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    geom = pointcloud.PointCloud(x=x_grid, y=x_grid, epsilon=sinkhorn_epsilon)
</pre><p>No need to pass <code>y=x_grid</code>,it's done by default if <code>y</code> is not passed.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>